### PR TITLE
updating r2s

### PR DESCRIPTION
**Pull request by moatazharb**
Yet to be tested. Don't merge yet.

### CODE REVIEW

File: scripts/r2s.py [hunk 1]
```diff
@@ -140,10 +140,10 @@ def step1():
     ves = list(mesh.iter_ve())
     tags_keep = ("cell_number", "cell_fracs",
                  "cell_largest_frac_number", "cell_largest_frac")
-    for tag in mesh.mesh.getAllTags(ves[0]):
+    for tag in mesh.get_all_tags():
         if tag.name not in tags_keep:
-            mesh.mesh.destroyTag(tag, True)
-    mesh.mesh.save('blank_mesh.h5m')
+            mesh.delete(tag)
```
Comment (zxkjack123) [inline scripts/r2s.py:145]: Error jumps out at this line:
```
Traceback (most recent call last):
  File "/root/.local/bin/r2s.py", line 223, in <module>
    main()
  File "/root/.local/bin/r2s.py", line 218, in main
    step1()
  File "/root/.local/bin/r2s.py", line 145, in step1
    mesh.delete(tag)
AttributeError: 'MeshTally' object has no attribute 'delete'
```

Comment (zxkjack123) [reply scripts/r2s.py:145]: Sorry, the delete tag function hasn't been merged. So, it of course has no attribute 'delete'.

Comment (gonuke) [reply scripts/r2s.py:145]: If only someone would review and merge it :)

**Comment by gonuke** [comment]
if you rebase this, you will probably find that it fails on line 129 because we removed the dagmc dependency from the `r2s` module

**Comment by gonuke** [comment]
rebase against `CNERG/pymoab_cleanup`

**Comment by zxkjack123** [comment]
The #48 rebased this PR on pymoab_cleanup, and fixed some errors.

**Comment by gonuke** [comment]
Can someone confirm that this PR is superseded by #48 and close this PR?

**Comment by moatazharb** [comment]
Closing this PR as the changes have been incorporated in #48 by @zxkjack123

**Pull request closed without merging**